paf-mvp-core-js/src/express/identity-endpoint.ts

```diff
@@ -5,17 +5,21 @@ import cors from 'cors';
 import { Express } from 'express';
 import { GetIdentityResponseBuilder } from '@core/model/identity-response-builder';
 
-export const addIdentityEndpoint = (
-  app: Express,
-  name: string,
-  type: 'vendor' | 'operator',
-  keys: KeyInfo[],
-  dpoEmailAddress: string,
-  privacyPolicyUrl: URL
-) => {
-  const response = new GetIdentityResponseBuilder(name, type, dpoEmailAddress, privacyPolicyUrl).buildResponse(keys);
+export const addIdentityEndpoint = (app: Express, identity: Identity) => {
+  const { name, type, currentPublicKey, dpoEmailAddress, privacyPolicyUrl } = identity;
+  const response = new GetIdentityResponseBuilder(name, type, dpoEmailAddress, privacyPolicyUrl).buildResponse([
+    currentPublicKey,
+  ]);
 
   app.get(participantEndpoints.identity, cors(corsOptionsAcceptAll), (req, res) => {
     res.send(response);
   });
 };
+
+export interface Identity {
+  name: string;
+  currentPublicKey: KeyInfo;
```

Comment:
FAR: KeyInfo isn't enough descriptive. PublicKeyInfo maybe.
OPT: Fall back to a list of KeyInfo as it was before.

Reply:
> FAR: KeyInfo isn't enough descriptive. PublicKeyInfo maybe.

Agreed. I'll take it in another PR

> OPT: Fall back to a list of KeyInfo as it was before.

I hesitate, but I think you're right. I will also fix it in a dedicated PR